projects/vdk-core/tests/vdk/internal/builtin_plugins/run/job_input_error_classifier_test.py

```diff
@@ -87,18 +98,10 @@ def test_unknown_generic_error(self, mock_is_user_error, mock_traceback_format_t
             errors.ResolvableBy.USER_ERROR,
         )
 
-    # Generic error thrown by job_input that is not specifically recognised by VDK should be VAC error.
-    @patch(f"{traceback.format_tb.__module__}.{traceback.format_tb.__name__}")
-    @patch(f"{is_user_error.__module__}.{is_user_error.__name__}")
-    def test_job_input_generic_error(
-        self, mock_is_user_error, mock_traceback_format_tb
-    ):
-        exception = Exception("!")
-        mock_is_user_error.return_value = False
-        mock_traceback_format_tb.return_value = self.PLATFORM_ERROR_STACKTRACE
```

Comment:
Why delete this? Where is the test covering  PLATFORM_ERROR_STACKTRACE ? It correctly test that "PLATFORM_ERROR_STACKTRACE" is a platform error.